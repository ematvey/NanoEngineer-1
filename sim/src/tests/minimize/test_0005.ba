LENGTH 0 1 0.770062335139
LENGTH 0 2 0.770013636243
LENGTH 0 3 1.55573294623
LENGTH 0 6 1.63041988457
LENGTH 9 10 0.768899213161
LENGTH 9 11 0.769969479915
LENGTH 9 12 1.75952777756
LENGTH 3 4 0.769252884297
LENGTH 3 5 0.77067502879
LENGTH 3 9 1.85955263437
LENGTH 12 13 0.769987662239
LENGTH 12 14 0.768966189114
LENGTH 6 7 0.769362073409
LENGTH 6 8 0.769749309841
LENGTH 6 12 1.54000194805
ANGLE 0 3 4 112.083612472
ANGLE 0 3 5 99.6214319304
ANGLE 0 3 9 109.704511138
ANGLE 0 6 7 115.257844633
ANGLE 0 6 8 101.203865489
ANGLE 0 6 12 120.826736622
ANGLE 1 0 2 109.397770219
ANGLE 1 0 3 108.588148475
ANGLE 1 0 6 112.942272293
ANGLE 2 0 3 111.421634557
ANGLE 2 0 6 110.280691994
ANGLE 3 0 6 104.131747411
ANGLE 3 9 10 113.964933553
ANGLE 3 9 11 114.375816288
ANGLE 3 9 12 101.726060986
ANGLE 4 3 5 109.423185971
ANGLE 4 3 9 107.545804666
ANGLE 5 3 9 118.390993049
ANGLE 6 12 9 103.304578082
ANGLE 6 12 13 109.826270378
ANGLE 6 12 14 111.914202775
ANGLE 7 6 8 109.391270139
ANGLE 7 6 12 97.9187517163
ANGLE 8 6 12 112.403674886
ANGLE 9 12 13 113.212594669
ANGLE 9 12 14 109.021124188
ANGLE 10 9 11 109.36368228
ANGLE 10 9 12 115.164498925
ANGLE 11 9 12 101.545698171
ANGLE 13 12 14 109.48016823
