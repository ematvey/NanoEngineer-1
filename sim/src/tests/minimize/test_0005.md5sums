stdout B5313F41C12B97E461CFE5F6E71B2C75
stderr D41D8CD98F00B204E9800998ECF8427E
