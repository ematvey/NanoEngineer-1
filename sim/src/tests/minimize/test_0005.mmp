mmpformat 080327 required; 080523 preferred
kelvin 300
group (View Data)
info opengroup open = True
csys (HomeView) (1.000000, 0.000000, 0.000000, 0.000000) (10.000000) (0.000000, 0.000000, 0.000000) (1.000000)
csys (LastView) (-0.421789, 0.882976, 0.067448, -0.194675) (5.845851) (0.000000, 0.000000, 0.000000) (1.000000)
egroup (View Data)
group (test_0005)
info opengroup open = True
mol (Chunk-C1) def
atom 1 (6) (-16, 344, -737) def
atom 2 (1) (-92, 1221, -108) def
bond1 1
atom 3 (1) (1, 648, -1775) def
bond1 1
atom 4 (6) (-1225, -588, -437) def
bond1 1
atom 5 (1) (-1813, -235, 400) def
bond1 4
atom 6 (1) (-1787, -456, -1352) def
bond1 4
atom 7 (6) (1288, -573, -395) def
bond1 1
atom 8 (1) (1869, -250, 459) def
bond1 7
atom 9 (1) (1874, -449, -1296) def
bond1 7
atom 10 (6) (-613, -2292, -13) def
bond1 4
atom 11 (1) (-1031, -2694, 900) def
bond1 10
atom 12 (1) (-724, -3017, -808) def
bond1 10
atom 13 (6) (1130, -2052, 4) def
bond1 7 10
atom 14 (1) (1574, -2219, 976) def
bond1 13
atom 15 (1) (1584, -2716, -719) def
bond1 13
egroup (test_0005)
end1
group (Clipboard)
info opengroup open = False
egroup (Clipboard)
end molecular machine part test_0005
