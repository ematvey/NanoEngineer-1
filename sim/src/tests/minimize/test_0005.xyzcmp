15
RMS=0.989600
C 0.032720 0.255172 -0.662529
H -0.047618 1.103362 0.016924
H 0.013723 0.653294 -1.676773
C -1.152093 -0.730916 -0.443916
H -1.803585 -0.371663 0.352365
H -1.758332 -0.803731 -1.346519
C 1.358032 -0.521390 -0.408654
H 1.921267 -0.057890 0.400956
H 1.990359 -0.495118 -1.295800
C -0.558378 -2.121752 -0.073753
H -0.930912 -2.448608 0.896762
H -0.862842 -2.871483 -0.803673
C 0.992710 -1.990445 -0.044816
H 1.377507 -2.244632 0.942551
H 1.446442 -2.682202 -0.754124
