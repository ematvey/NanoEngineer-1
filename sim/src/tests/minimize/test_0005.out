======= test_0005.test =======
TYPE struct
======= exitvalue =======
0
======= structurematch =======
1
======= lengthsangles =======
OK
======= stderr =======
cutover to fine tolerance at 15
== structure comparison ==
cutover to fine tolerance at 1
